repose-aggregator/components/services/opentracing-service/opentracing-service-impl/build.gradle

```diff
@@ -0,0 +1,33 @@
+dependencies {
+    compile project(":repose-aggregator:commons:commons-utilities")
+    compile project(":repose-aggregator:core:repose-core-api")
+    compile project(":repose-aggregator:components:services:opentracing-service:opentracing-service-api")
+    compile "org.slf4j:slf4j-api"
+    compile "org.slf4j:jcl-over-slf4j"
+    compile "org.springframework:spring-beans"
+    compile "io.opentracing:opentracing-api:0.31.0"
```

Comment:
Shouldn't be specifying versions in any of your build files

Reply:
yup!  good catch.  I specified it already in the properties so I'll remove all this.